### PR TITLE
fix moar flaky StorageTest.testNotificationStorage() [FINERACT-828]

### PR DESCRIPTION
**Pull request by vorburger**
similar fix to earlier Pull Request for the same

also added some more toString()

see FINERACT-828

### CODE REVIEW

**Comment by vorburger** [comment]
@thesmallstar @xurror review this?

File: fineract-provider/src/main/java/org/apache/fineract/useradministration/domain/AppUser.java [hunk 1]
```diff
@@ -663,4 +663,8 @@ public boolean isSelfServiceUser() {
         return newAppUserClientMappings;
     }
 
+    @Override
+    public String toString() {
```
Comment (xurror) [inline fineract-provider/src/main/java/org/apache/fineract/useradministration/domain/AppUser.java:667]: Just curious, I'm not seeing this method being used anywhere, I wish to know why you wrote it

Comment (thesmallstar) [reply fineract-provider/src/main/java/org/apache/fineract/useradministration/domain/AppUser.java:667]: Hello @xurror, if I am not wrong, it is explicitly used for debugging(if the test fails). 
Refer to this how it helped: #858  and https://issues.apache.org/jira/browse/FINERACT-828

**Review by thesmallstar — APPROVED** [review]
LGTM

**Pull request merged**